modules/vault/tests/test_vault.py

```diff
@@ -0,0 +1,39 @@
+from testcontainers.vault import VaultContainer
+
+
+def test_docker_run_vault():
+    config = VaultContainer()
```

Comment:
For our tests we want to pin the vault version to a specific tag in order to have reproducible tests. Could you pin the image version to `VaultContainer('hashicorp/vault:1.16.1')`? 

The `latest` tag in the default constructor argument can stay as-is.